### PR TITLE
disable loading on empty ped file submisison

### PR DESCRIPTION
**Pull request by hanars**
## Pull request overview

This PR prevents AnVIL workspace data loading requests from proceeding when the uploaded pedigree file contains no sample rows (e.g., header-only), returning a clear validation error instead.

**Changes:**
- Add validation in `_validate_expected_samples` to error when no pedigree samples are present.
- Add a regression test ensuring `add_workspace_data` returns a 400 with the new error on an empty pedigree upload.

### CODE REVIEW

File: seqr/views/apis/anvil_workspace_api.py [hunk 1]
```diff
@@ -226,6 +226,9 @@ def _validate_expected_samples(vcf_samples, loaded_sample_types, loaded_individu
         else:
             errors.append('New data cannot be added to this project until the previously requested data is loaded')
 
+    if not record_family_ids:
+        errors.append('No samples found in the pedigree file')
+
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/anvil_workspace_api.py:231]: that test does not have any erorr mode testing and adding it feels out of scope

**Review by larrybabb — APPROVED** [review]
super straight forward. +1
